### PR TITLE
Remove fast log exp and pow methods introduced in #992

### PR DESCRIPTION
**Pull request by MaxGraey**
Due to some precision regressions in new fast exp_lut, log_lut and pow_lut methods I temporary disabled them until I find out if there are general implementation problems or just some errors made me during the porting process.

### CODE REVIEW

**Comment by MaxGraey** [comment]
That's how issue looks like:

precision-lost-example

Actually it appear in with double logarithm and pretty big arguments

**Comment by MaxGraey** [comment]
I reproduced this with original musl library so can confirm it implementation issue. We can't use this functions.

@dcodeIO plz merge this asap

File: tests/compiler/std/math.ts [hunk 1]
```diff
@@ -1538,7 +1538,7 @@ assert(test_expm1f(NaN, NaN, 0.0, 0));
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 // Math.exp2
 ////////////////////////////////////////////////////////////////////////////////////////////////////
-
+/*
```
Comment (dcodeIO) [inline tests/compiler/std/math.ts]: With exp2 removed, should this be removed as well?

Comment (MaxGraey) [reply tests/compiler/std/math.ts]: I think leave it. May be in future I implement it again and this tests will be necessary

Comment (dcodeIO) [reply tests/compiler/std/math.ts]: ok :)

**Comment by MaxGraey** [comment]
Live demo: https://maxgraey.github.io/mandelbrot-issue-demo
Repo: https://github.com/MaxGraey/mandelbrot-issue-demo

**Comment by MaxGraey** [comment]
Issue was solved

**Pull request closed without merging**